scipy/optimize/rectangular_lsap/rectangular_lsap.cpp

```diff
@@ -116,55 +116,55 @@ augmenting_path(int nc, std::vector<double>& cost, std::vector<double>& u,
     return sink;
 }
 
-static int
-solve(int nr, int nc, double* input_cost, int64_t* output_col4row)
+static intptr_t
```

Comment:
This `int` is fine as-is.
The return code is either 0 for success or -1 for failure.

Reply:
Fixed in latest push